packages/@o3r/schematics/src/rules/install.ts

```diff
@@ -1,17 +1,21 @@
-import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import type { Rule } from '@angular-devkit/schematics';
 import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
 import { lastValueFrom } from 'rxjs';
-import { getPackageManager } from '../utility/package-manager-runner';
+import { getPackageManager, type SupportedPackageManagers } from '../utility/package-manager-runner';
 
 /**
  * Install the Otter packages
+ * @deprecated use {@link setupDependencies} instead, will be removed in V11
```

Comment:
removed in V12 I believe

Reply:
nop, we will not wait 1 year to remove a schematic helper